cloud/shared/bin/lib/terraform.py

```diff
@@ -1,37 +1,95 @@
 import subprocess
 import os
+import re
 import shutil
 import shlex
+import inspect
 from typing import Optional
 
 from cloud.shared.bin.lib.config_loader import ConfigLoader
 from cloud.shared.bin.lib.print import print
 
 
-# TODO(#2741): When using this for Azure make sure to setup backend bucket prior to calling these functions.
-def perform_apply(
+def force_unlock(
         config_loader: ConfigLoader,
-        is_destroy=False,
+        lock_id: str,
         terraform_template_dir: Optional[str] = None):
-    '''Generates terraform variable files and runs terraform init and apply.'''
     if not terraform_template_dir:
         terraform_template_dir = config_loader.get_template_dir()
-    tf_vars_filename = config_loader.tfvars_filename
 
-    terraform_cmd = f'terraform -chdir={terraform_template_dir}'
+    perform_init(config_loader, terraform_template_dir, False)
```

Comment:
**nitpick:** either add an inline comment hinting at what the boolean is or use an enum param instead of a boolean